src/reaper_osara.cpp

```diff
@@ -3762,6 +3776,57 @@ void cmdToggleLoopSegScrub(Command* command) {
 	Main_OnCommand(command->gaccel.accel.cmd, 0);
 }
 
+void cmdReportRegionMarkerItems(Command* command) {
+	const bool multiLine = lastCommandRepeatCount == 1;
+	ostringstream s;
+	auto separate = [&s, multiLine]() {
+		if(s.tellp()) {
+			s << (multiLine ? "\r\n" : ", ");
+		}
+	};
+	double pos = GetPlayState()? GetPlayPosition():GetCursorPosition();
+	double start,end;
+	bool isrgn;
+	int number;
+	const char* name{nullptr};
+	int idx = 0;
+	while(EnumProjectMarkers(idx++, &isrgn, &start, &end, &name, &number)>0) {
+		if(!(isrgn && start <= pos && pos <= end)) {
+			continue;
+		} 
+		separate();
+		if(name[0]) {
+			s << name ;
+		} else {
+			// Translators: used to report an unnamed region. {} is replaced with the region number.  
+			s << format(translate("region {}"), number);
+		}
+	}
+	int markerIdx;
+	GetLastMarkerAndCurRegion(nullptr, pos, &markerIdx, nullptr);
+	if(markerIdx >=0) {
+		EnumProjectMarkers(markerIdx, nullptr, nullptr, nullptr, &name, &number);
+		separate();
+		if (name[0]) {
+			s << name;
+		} else {
+			// Translators: used to report an unnamed marker. {} is replaced with the marker number.  
+			s << format(translate("marker {}"), number);
+		}
+	}
+	separate();
+	s << formatItemsWithState([pos](MediaItem* item) -> bool{
+		MediaTrack* track = GetMediaItem_Track(item);
+		return (isTrackSelected(track) && isPosInItem(pos, item));
```

Comment:
I understand why we only report items on selected tracks. Otherwise, we could report many, many items. However, I worry that this might not be obvious. I don't know if it's feasible to communicate this in the action name, but I wonder if we should at least note it in the readme.

Reply:
> I don't know if it's feasible to communicate this in the action name
> 
How's about OSARA: Report regions, closest project marker and items on selected tracks at current position

Reply:
@ScottChesworth  yes, the only thing is the marker is the nearest marker before the current position. What about "OSARA: Report regions, last project marker and items on selected tracks at current position"?

Reply:
> What about "OSARA: Report regions, last project marker and items on selected tracks at current position"?
> 
That's got it :)